modules/default.nix

```diff
@@ -1,5 +1,8 @@
 { lib, pkgs, ... }:
 
+let
+  colorizer = if lib.versionAtLeast lib.version "25.05pre" then "colorizer" else "nvim-colorizer";
```

Comment:
Are there guarantees that people are not using the default branch of nixvim with the stable version of nixpkgs?

Reply:
It did it myself until very recently because nixvim did not yet branch off back then.

Reply:
I think people should be using the matching nixpkgs version otherwise packages might not match and be missing.